packages/cli/src/build.ts

```diff
@@ -34,18 +35,21 @@ export default async function build(config, targetDir, options) {
           const rawFiles = await filesystem.promises.readFile(String(filePath), {
             includeConflicts: true
           });
-          const rawFile = rawFiles[0];
+          let rawFile = rawFiles[0];
           if (rawFiles.length > 1) {
-            // TODO: We could join colliding JSON files together. e.g. sidebar.json files
-            // if (path.basename(String(filePath)).startsWith('.') && path.extname(String(filePath)) === '.json') {
-            //   console.warn(`'${filePath}' returned multiple files at the same location. Since the files were JSON dot files, they were merged using a basic Lodash \`merge\`.`);
-            //   rawFile = Buffer.from(JSON.stringify(merge(...rawFiles.map(fileData => JSON.parse(String(fileData))))));
-            // } else {
-            throw new Error(`'${filePath}' returned multiple files at the same location, this will result in overwritten data in the output. Aborting build.
+            // Join colliding JSON files together. e.g. sidebar.json files
+            if (path.extname(String(filePath)) === '.json') {
+              console.warn(
+                `'${filePath}' returned multiple files at the same location. Since the files were JSON dot files, they were merged using a basic Lodash \`merge\`.`
```

Comment:
is a basic lodash merge enough?  We use `deepMerge` already in the config file....could we use that here?

Reply:
Oops - outdated comment. Fixed now